spec/services/create_next_edition_service_spec.rb

```diff
@@ -0,0 +1,118 @@
+RSpec.describe CreateNextEditionService do
+  describe ".call" do
+    let(:user) { create :user }
+
+    before do
+      populate_default_government_bulk_data
+    end
+
+    it "aborts if the current edition isn't live" do
+      current_edition = create(:edition, live: false)
+
+      expect { described_class.call(current_edition: current_edition, user: user) }
+        .to raise_error("Can only create a next edition from a live edition")
+    end
+
+    it "updates the documents current edition" do
+      current_edition = create(:edition, :published)
+
+      next_edition = described_class.call(current_edition: current_edition,
+                                          user: user)
+
+      expect(next_edition.current).to be(true)
+    end
+
+    it "returns a draft edition" do
+      current_edition = create(:edition, :published)
+
+      next_edition = described_class.call(current_edition: current_edition,
+                                          user: user)
+
+      expect(next_edition.draft?).to be(true)
+    end
+
+    it "resets change_note, update_type, and propoposed_publish_time values" do
+      current_edition = create(:edition, :published)
```

Comment:
It would be good to set change, note update type and proposed publish time on this so that it's clear that something actually changed

Reply:
@kevindew Yup, makes sense.  Will fix.